code/web/src/modules/user/api/actions.js

```diff
@@ -36,7 +36,7 @@ export function login(userCredentials, isLoading = true) {
     return axios.post(routeApi, query({
       operation: 'userLogin',
       variables: userCredentials,
-      fields: ['user {name, email, role}', 'token']
+      fields: ['user {name, email, id, role}', 'token']
```

Comment:
Adding in id to user to make our requests updating the user's style easier later